components/collective-page/sections/Budget.js

```diff
@@ -19,6 +19,7 @@ import StyledCard from '../../StyledCard';
 import { P, Span } from '../../Text';
 import { transactionsQueryCollectionFragment } from '../../transactions/graphql/fragments';
 import TransactionsList from '../../transactions/TransactionsList';
+import { withUser } from '../../UserProvider';
```

Comment:
There's also the hook variant `useUser` you can use if a parent component already provides the context.